node/src/main/scala/coop/rchain/node/api/DeployGrpcServiceV1.scala

```diff
@@ -291,5 +295,21 @@ object DeployGrpcServiceV1 {
             }
           )
           .flatMap(Observable.fromIterable)
+
+      def status(request: com.google.protobuf.empty.Empty): Task[StatusResponse] =
+        (for {
+          address <- RPConfAsk[F].ask
+          peers   <- ConnectionsCell[F].read
+          nodes   <- NodeDiscovery[F].peers
```

Comment:
An easy extend on this method can implement https://github.com/rchain/rchain/issues/3539

Reply:
Since we add this, why not make it complete?

Reply:
@zsluedem Are you suggesting to add complete peer table in status response? This can be a big list so better to have separate method.